webtranslate/users/development.py

```diff
@@ -3,8 +3,92 @@
 
 In particular the user management is very poor.
 """
+import os, configparser
 from webtranslate import rights, userauth
-from webtranslate.users import silly
+
+# Table with user names and plain text passwords.
+_users = None
+
+FILENAME = "users.dat"
+
+def init_users():
+    """
+    Initialize the user authentication system.
+    """
+    global _users
+
+    _users = set()
+    if not os.path.isfile(FILENAME): return
+
+    handle = open(FILENAME, "r", encoding = "utf-8")
+    for line in handle:
+        line = line.rstrip()
+        if len(line) == 0 or line[0] == '#': continue
+        i = line.find(':')
+        if i < 0: continue
+        j = line.find(':', i + 1)
+        if j < 0:
+            j = len(line)
+        uname, pwd = line[:i].strip(), line[i+1:j]
+
+        if len(uname) == 0 or len(pwd) == 0: continue
+        _users.add((uname, pwd))
+
+    handle.close()
+
+def authenticate(user, pwd):
+    """
+    Is the provided user and password combination valid?
+
+    @param user: User name.
+    @type  user: C{str}
+
+    @param pwd: Password.
+    @type  pwd: C{str}
```

Comment:
`pwd` .. not only is it lazy, it also is misleading ;) Why not just `password`?

Reply:
i am just moving code around to isolate the horrors of the development auth system from the rest.. please make PR if you want to spend time on the development authentication.